src/plugins.rs

```diff
@@ -18,6 +18,6 @@ lazy_static! {
         #[cfg(feature = "plugin-oob_messages")]
         Arc::new(Mutex::new(oob_messages::plugin::OOBMessages {})),
         #[cfg(feature = "plugin-didcomm_messaging")]
-        Arc::new(Mutex::new(didcomm_messaging::plugin::MediatorCoordination::default())),
+        Arc::new(Mutex::new(didcomm_messaging::plugin::DidcommMessaging::default())),
     ];
 }
```

Comment:
Yes, it is. Yes, but it was not handled in this file. I saw it while working on this ticket and decided to modify it since I was the one who worked on that ticket.